test/api/transactions.js

```diff
@@ -0,0 +1,183 @@
+'use strict'; /*jslint mocha:true, expr:true */
+
+
+var node = require('./../node.js');
+
+var params = {
+    blockId: '7807109686729042739',
+    transactionId: '16733264093386669800',
+    address: '16009998050678037905L',
+    offset: 20,
+    limit: 100
+};
+
+describe('Transactions API', function() {
+
+    /*Define functions for use within tests*/
+    function getTransaction(id, done) {
+        node.get('/api/getTransaction?transactionId=' + id, done);
+    }
+
+    function getUnconfirmedTransactions(done) {
+        node.get('/api/getUnconfirmedTransactions', done);
+    }
+
+    function getLastTransactions(done) {
+        node.get('/api/getLastTransactions', done);
+    }
+
+    function getTransactionsByAddress(id, id2, id3, done) {
+        node.get('/api/getTransactionsByAddress?address=' + id + '&offset=' + id2 + '&limit=' + id3, done);
+    }
+
+    function getTransactionsByBlock(id, id2, id3, done) {
+        node.get('/api/getTransactionsByBlock?blockId=' + id + '&offset=' + id2 + '&limit=' + id3, done);
+    }
+
+    function checkTransactionsBody(id) {
+        for (var i = 0; i < id.length; i++) {
+            if (id[i + 1]) {
+                node.expect(id[i]).to.have.any.keys(
```

Comment:
test fails with all keys because knowlege only works on delegates and known accounts.